src/util/checkIsOnChain.ts

```
@@ -4,13 +4,23 @@
 		case ProposalType.BOUNTIES:
 			return Boolean(await api.query.bounties.bounties(Number(id)));
 		case ProposalType.CHILD_BOUNTIES:
-			return Boolean(await api.query.childBounties.childBounties(Number(id)));
+			if (isChildBountyId(id)) {
+				const [parentBountyId, childBountyId] = id;
+				return Boolean(await api.query.childBounties.childBounties(parentBountyId, childBountyId));
+			}
+			return false;
 		case ProposalType.DEMOCRACY_PROPOSALS:
 			dataFromChain = ((await api.query.democracy.publicProps()).toJSON() as any[]) || [];
 			return Boolean(dataFromChain.filter(([id]) => id === id).length);
```

Biome (1.9.4)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)